libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -281,6 +291,18 @@ typedef struct _mqttConnection
     size_t pingreqPacketSize;                    /**< @brief The size of an allocated PINGREQ packet. */
 } _mqttConnection_t;
 
+/**
+ * @brief Represents a mapping of MQTT Connection to the corresponding MQTT context
+ * which used to call MQTT v4 beta_2 API that contains all the information on network
```

Comment:
Please call out the versions of the library for MQTT connection and MQTT context and improve the comments to make it more readable.